src/main/flight/autopilot.c

```diff
@@ -91,17 +183,175 @@ void altitudeControl(float targetAltitudeCm, float taskIntervalS, float vertical
     DEBUG_SET(DEBUG_AUTOPILOT_ALTITUDE, 7, lrintf(altitudeF));
 }
 
-bool isBelowLandingAltitude(void)
+void setSticksActiveStatus(bool areSticksActive)
 {
-    return getAltitudeCm() < 100.0f * autopilotConfig()->landing_altitude_m;
+    posHold.sticksActive = areSticksActive;
+}
+
+void setTargetLocation(gpsLocation_t newTargetLocation)
+{
+    currentTargetLocation = newTargetLocation;
+    // note: will cause a large D and A spike if there is an abrupt change in distance
+    // function is intended for only small changes in position
+    // for example, where the step distance change reflects an intended velocity, determined by a client function
+    // if we had a 'target_ground_speed' value, like in gps_rescue, we can make a function that starts and stops smoothly and targets that velocity
 }
 
-const pidCoefficient_t *getAltitudePidCoeffs(void)
+void resetLocation(earthFrame_t *efAxis, axisEF_t loopAxis)
 {
-    return &altitudePidCoeffs;
+    if (loopAxis == EW) {
+        currentTargetLocation.lon = gpsSol.llh.lon; // update East-West / / longitude position
+    } else {
+        currentTargetLocation.lat = gpsSol.llh.lat; // update North-South / latitude position
+    }
+    efAxis->previousDistance = 0.0f; // and reset the previous distance
+}
+
+bool positionControl(void) {
```

Comment:
See https://betaflight.com/docs/development/CodingStyle#curly-braces
```suggestion
bool positionControl(void)
{
```

Reply:
thanks, apologies for missing your comment